### PR TITLE
You can now emag the escape pods to launch them

### PR DESCRIPTION
**Pull request by Core0verload**
Now you can emag the control panel, press the button and enjoy your trip to lavaland.

This adds a new fun way to get rid of targets, and makes it possible to sabotage pods and force people to use the shuttle.

Complimentary changes:
* The pod storage will permanently unlock when the pod is launched. This fixes a corner case when the pod is launched in Code Red, but then alert level gets dropped and storage stops working.
* The pod storage will now also open on click, just like it does on click-drag.

:cl: CoreOverload
add: You can now emag the escape pods to launch them under any alert code.
/:cl:

### CODE REVIEW

File: code/modules/shuttle/emergency.dm [hunk 1]
```diff
@@ -387,7 +387,12 @@
 	launch_status = UNLAUNCHED
 
 /obj/docking_port/mobile/pod/request()
-	if(security_level == SEC_LEVEL_RED || security_level == SEC_LEVEL_DELTA)
+	var/emagged = FALSE
+	var/obj/machinery/computer/shuttle/S = getControlConsole()
+	if(S)
+		emagged = S.emagged
+
+	if(security_level == SEC_LEVEL_RED || security_level == SEC_LEVEL_DELTA || emagged)
```
Comment (RemieRichards) [inline code/modules/shuttle/emergency.dm]: could just be "```|| S && S.emagged```, no need for a local.

**Comment by Cyberboss** [comment]
Add picks to the pods when?

**Comment by Fox-McCloud** [comment]
The problem I have with this is..when would you never *not* emag the pods (if you get an emag)?

**Comment by Metallusions** [comment]
@Cyberboss emergency disembarkation tools are already available in pod storage.

**Comment by Hatfish** [comment]
Pod launching should probably get a delay so traitors don't have to come along for the ride if they want to dispose of someone.

**Comment by Core0verload** [comment]
It has a delay, just enough to leave the pod with two seconds spare. If you are too slow - well, you are in for a fuuun ride.

**Pull request merged**